debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OracleConnectorIT.java

```diff
@@ -2136,4 +2143,121 @@ private void verifyHeartbeatRecord(SourceRecord heartbeat) {
     private long toMicroSecondsSinceEpoch(LocalDateTime localDateTime) {
         return localDateTime.toEpochSecond(ZoneOffset.UTC) * MICROS_PER_SECOND;
     }
+
+    @Test(expected = DebeziumException.class)
```

Comment:
As a nit-pick, could you please annotate all your new test methods with `@FixFor("DBZ-3986")`.